README.md

```diff
@@ -49,7 +49,7 @@ core23_shariff:
 ```twig
 {# template.twig #}
 
-{{ sonata_block_render({ 'type': 'core23_shariff.block.share' }, {
+{{ sonata_block_render({ 'type': 'core23.shariff.block.share' }, {
```

Comment:
The block name was changed with the last commits: https://github.com/core23/ShariffBundle/commit/6975b8cb990f3179e8f9789953befcf6b0105efc#diff-d1df843008929063a56808f139bb9ae7R7

Reply:
Sorry, didn't realize that and thought it was a typo in the readme -_-.

Reply:
Thanks for your help anyway.

An improvement for the `sonata_block` config is still welcome :)